src/python/tests/core/test_scheduler_get_tasks.py

```diff
@@ -25,29 +25,31 @@
 14. LIMIT  + OFFSET + WHERE + SORT
 """
 
-import aiopg
 import asyncio
 import datetime
 import os
 import random
-import sqlalchemy
-import sqlalchemy.dialects.postgresql
+import signal
+import sys
 import time
 import uuid
-
+import aiopg
```

Comment:
We should aim to remove this from tests so that the tests will work with any database and use the storage service